Activities/Database/UiPath.Database.Activities/Properties/UiPath.Database.Activities.resx

```diff
@@ -417,7 +417,7 @@
     <comment>activity name</comment>
```

Comment:
Please follow the naming rules https://uipath.atlassian.net/wiki/spaces/ACTVCGD/pages/87179755688/Official+Activities+UX+Design+Guidelines#Activities-naming-rules

Several resource texts need to be updated (activity name and field names) -- see the latest version of the specs document.